datasets/ONS-UK-trade-in-services/main.py

```diff
@@ -1,58 +1,196 @@
 #!/usr/bin/env python
 # coding: utf-8
 # %%
+from gssutils import *
+import json
+
+cubes = Cubes("info.json")
+scraper_1 = Scraper('https://www.ons.gov.uk/businessindustryandtrade/' + \
+                  'internationaltrade/datasets/uktradeinservicesallcountriesnonseasonallyadjusted')
+scraper_1
+trace = TransformTrace()
+
+# %%
+info = json.load(open("info.json"))
+scraper_1.dataset.family = info["families"]
+
+# %%
+dist_1 = scraper_1.distributions[0]
+dist_1
+
+# %%
+first_url = dist_1.downloadURL
+first_url
+
+# %%
+tabs = {tab.name: tab for tab in scraper_1.distribution(latest=True, mediaType=Excel).as_databaker()}
+
+
+# %%
+tab = tabs["TiS by country"]
 
+# %%
+tab = tabs['TiS by country']
+datasetTitle = "ONS-UK-trade-in-services"
+columns = ["Period", "ONS Partner Geography", "Flow", "Trade Services", "Measure Type", "Unit", "Marker"]
+
+# %%
+trace.start(datasetTitle, tab, columns, first_url)
+
+# trace.obs("Taken from cell C7 across and down which are non blank")
+observations = tab.excel_ref('C7').expand(DOWN).expand(RIGHT).is_not_blank()
+
+trace.Period("Taken from cell C4 across")
+period = tab.excel_ref('C4').expand(RIGHT).is_not_whitespace()
+
+trace.Flow("Taken from cells B5 and B252")
+flow = tab.excel_ref('B').expand(DOWN).by_index([5,252])
+
+footer = tab.excel_ref("A500").expand(RIGHT).expand(DOWN)
 
+trace.Trade_Services("Hardcoded as all")
+
+trace.ONS_Partner_Geography("Taken from cell A7 down")
+ons_partner_geography = tab.excel_ref('A7').expand(DOWN).is_not_blank() - footer
+
+dimensions = [
+            HDim(period, 'Period', DIRECTLY,ABOVE),
+            HDim(ons_partner_geography, 'ONS Partner Geography', DIRECTLY,LEFT),
+            HDim(flow, 'Flow', CLOSEST,ABOVE),
+            HDimConst("Trade Services", "all"),
+           ]
+tidy_sheet = ConversionSegment(tab, dimensions, observations)
+savepreviewhtml(tidy_sheet, fname = tab.name + "Preview.html")
+trace.with_preview(tidy_sheet)
+trace.store("combined_dataframe", tidy_sheet.topandas())
+
+# %%
+#scraped second landing page
 from gssutils import *
+import json
+
+# %%
+info = json.load(open("info.json"))
+scraper_2 = Scraper(seed="info.json")
+scraper_2
 
+# %%
+dist_2 = scraper_2.distributions[0]
+dist_2
+
+# %%
+second_url = dist_2.downloadURL
+second_url
+
+# %%
+tabs = {tab.name: tab for tab in scraper_2.distribution(latest=True, mediaType=Excel).as_databaker()}
+
+# %%
+tab = tabs['Time Series']
+
+datasetTitle = 'UK trade in services by partner country'
+columns=["Period", "Flow", "Trade Services", "ONS Partner Geography", "Marker", "Seasonal Adjustment"]
+trace.start(datasetTitle, tab, columns, second_url)
```

Comment:
replace line 93 with :

`trace.start(datasetTitle, tab, columns, distribution.downloadURL)`

Reply:
Variable holding distribution download is removed and changes made to trace.start()